src/views/Home/components/FAQ/index.scss

```diff
@@ -0,0 +1,78 @@
+@use "./stylesheets/responsiveness";
+@use "./stylesheets/typography";
+
+.faq-component {
+  margin-top: -1px;
+  background-size: cover;
+  background: linear-gradient(
```

Comment:
I'd recommend pulling in the new changes since there's a new footer. I just want to see how that looks distance wise given the new footer.